root/static/scripts/guess-case/MB/Control/GuessCase.js

```diff
@@ -60,7 +61,7 @@ var mode = ko.computed({
 
         if (modeName !== gc.modeName) {
             gc.modeName = modeName;
-            gc.mode = MB.GuessCase.Mode[modeName];
+            gc.mode = require('../../modes')[modeName];
```

Comment:
I assume this points to the *root/static/scripts/guess-case/modes.js* script. I'm a bit confused about what `require('../../modes')` function returns and what's referenced by `modeName`.

Is there a good summary of new features out there? I'm always a bit behind on all new JS/node stuff. :(